tools/ui/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "@dfinity/agent": "0.8.4",
+    "@dfinity/principal": "0.9.0",
```

Comment:
This will probably break the UI code. Let's either fix the import, or keep the version at `0.8.9`

Reply:
Ah you're right. I'll fix the other references so UI is up to date with the new 0.9.0 imports